tests/lib/tools/suite/system-state/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: smoke test for the system-state tool
+
+execute: |
+
+    # Check help
+    "$TESTSTOOLS"/system-state | MATCH "usage: is-core"
+    "$TESTSTOOLS"/system-state -h | MATCH "usage: is-core"
+    "$TESTSTOOLS"/system-state --help | MATCH "usage: is-core"
+
+    # check ubuntu core 16
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+        "$TESTSTOOLS"/system-state is-core
+        "$TESTSTOOLS"/system-state is-core-16
+        ! "$TESTSTOOLS"/system-state is-core-18
```

Comment:
This is incorrect, please use `not`, instead.
`!` is only safe to use if it is the only and last element of a function or program.